src/reaktoro_pse/core/reaktoro_outputs.py

```diff
@@ -457,7 +458,10 @@ def scalingTendency(self, property_index):
         jsp = thermo_model.params().dumpJson()
         jsp_dict = json.loads(jsp)
         not_implemented = False
-        if isinstance(jsp_dict, list):
+        # TODO: need to add pE calculation to be able to calc scaling tendcies for these props.
+        if isinstance(jsp_dict, list) and "e-" not in [
+            s.name() for s, _ in reactant_species
```

Comment:
So the assumption here is that for pE calculations, "e-" will be present in the names? Is that always the case?